2014-15/D01 Make Individual File.sps

```diff
@@ -410,14 +411,19 @@ Else if (recid = "HC").
     Compute HC_episodes = 1.
 
     * Hours count.
-    Compute HC_total_hours = hc_hours.
+    Compute HC_total_hours = hc_hours_q1 + hc_hours_q2 + hc_hours_q3 + hc_hours_q4.
```

Comment:
Need to replace this bit with `hc_hours_annual` once that's been updated/added in the ep file.